.drone.star

```diff
@@ -1088,7 +1088,7 @@ def uiTests(ctx):
 
     # For ordinary PRs, always run the "minimal" UI test pipeline
     # That has its own expected-failures file, and we always want to know that it is correct,
-    if (ctx.build.event != "tag"):
+    if (ctx.build.event != "tag" and ctx.build.event != "cron" and "full-ci" not in ctx.build.title.lower()):
```

Comment:
`!= "cron"` will stop these from being run in the nightly "cron" CI job - that is a requirement here, good.

`"full-ci" not in ctx.build.title.lower()` means that if someone puts `full-ci` in their PR title, then these smoke test pipelines will not run. It is possible that the code in a PR has changed something that makes a smoke test fail, but we won't find that out. I think that will be surprising, when one day a PR has green CI, gets merged, and then the smoke tests in the master branch are failing.

Reply:
Currently, only smoke webUI tests run in normal PRs (without `full-ci`). If the PR has `full-ci` then `all + smoke` webUI tests pipelines will run which is redundant.
So, we wanted to not run the tests redundantly. Only run all webUI pipeline or smoke webUI pipeline
Same goes for nightly

Reply:
Do whatever is the requirement.

Reply:
OK, I think that I understand now. The "smoke tests" are maybe just test scenarios in within the ordinary test suites that are tagged as smoke tests. In that case, it is redundant to run them twice.